modules/021-cni-cilium/openapi/config-values.yaml

```diff
@@ -63,6 +63,11 @@ properties:
     default: false
     description: |
       Do not enforce any Network Policy. Just log allowed/disallowed connections.
+  cniExclusive:
+    type: boolean
+    default: true
+    description: |
+      Determines whether to remove other CNI configuration files for kubelet on nodes. If the Istio module's [`dataPlane.trafficRedirectionSetupMode` setting](../istio/configuration.html#parameters-dataplane-trafficredirectionsetupmode) is set to `CNIPlugin`, the option will be automatically set to `false`.
```

Comment:
This is unclear how this action will affect the cluster. Is it safe to disable? When do I need to disable this flag.

Reply:
```suggestion
      Prevents additional CNI plugins from being enabled on cluster nodes. If the Istio module's [`dataPlane.trafficRedirectionSetupMode` setting](../istio/configuration.html#parameters-dataplane-trafficredirectionsetupmode) is set to `CNIPlugin`, the option will be automatically set to `false`.
```

Reply:
I think, it is more clear.